package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Ubie Design Tokens",
   "style": "dist/tokens.css",
   "main": "dist/tokens.js",
-  "typings": "design-tokens.d.ts",
+  "typings": "dist/tokens.d.ts",
   "files": [
     "dist/**",
     "design-tokens.d.ts"
```

Comment:
Please delete this line
@yukukotani

Reply:
@takanorip nice catch! [395c491](https://github.com/ubie-oss/design-tokens/pull/10/commits/395c491b8ac1b86dc633bda0be213a889af4fe14)